src/ctapipe/calib/camera/calibrator.py

```diff
@@ -244,12 +244,6 @@ def _calibrate_dl1(self, event, tel_id):
             waveforms -= np.atleast_2d(dl1_calib.pedestal_offset)[..., np.newaxis]
 
         if n_samples == 1:
-            # To handle ASTRI and dst
```

Comment:
I think this one still applies (see #1777 )

Reply:
Do we still want to support DSTs? if so, should that be written here as a TODO? I would expect it to be an issue, but nothing in the code.